src/RenderWebGL.js

```diff
@@ -29,10 +29,15 @@ const MAX_TOUCH_SIZE = [3, 3];
  * target is touching a color whose components are each within this tolerance of
  * the corresponding component of the query color.
  * between 0 (exact matches only) and 255 (match anything).
- * @type {int}
+ * @type {object.<string,int>}
  * @memberof RenderWebGL
  */
-const TOLERANCE_TOUCHING_COLOR = 2;
+const TOLERANCE_TOUCHING_COLOR = {
+    R: 7,
+    G: 7,
+    B: 15,
```

Comment:
Why does Scratch 2.0 specify a different tolerance mask for the blue component (`0xf0`) vs. red & green (`0xf8`)?

Unfortunately the project in the bug report depends on the extra blue tolerance.  The lines are drawn with `#000000` but the code checks for `#00000d`.

Reply:
To be honest I'm not certain why Scratch 2.0 works this way. I do know that the typical human eye is better at distinguishing minor changes in shades of green than in red or blue, so it might be related to that.